packages/checkbox/src/vaadin-checkbox.js

```diff
@@ -48,17 +42,11 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  * @fires {CustomEvent} indeterminate-changed - Fired when the `indeterminate` property changes.
  *
  * @extends HTMLElement
- * @mixes ControllerMixin
```

Comment:
No need to apply `ControllerMixin` as it's already used by `LabelMixin` internally.